mergify_engine/engine/actions_runner.py

```diff
@@ -151,6 +189,18 @@ def _has_merge_action_with_strict_mode(
     )
 
 
+def _has_merge_action_with_commit_message_mode(
+    pull_request_rules: rules.PullRequestRules,
+) -> str:
+    nameValue = ""
+    for rule in pull_request_rules:
+        for name, action in rule.actions.items():
+            if name in ("merge", "queue") and ("commit_message" in action.raw_config):
+                nameValue = name
+                return nameValue
+    return ""
```

Comment:
Why are you returning an `str` here?
You could just return a `bool` that would be used in the code below.

Reply:
Because i wanted to do a verification when i call this function, to know if the action is merge or queue and add to the summary the right message. Or maybe i can use the same message of deprecation for both, but i think it can be confusing :/

Reply:
Alright, that makes sense. If you want to include the name of the action though, it's only in one place in your message so it'd be easy to use something like `{action_name}` in your deprecation message and replace it on the fly.

In the case of this function, you should not remove an empty string — it's always a bad design / idea.  Just return `None` to signal that you didn't find a result.